screens/Map.js

```diff
@@ -95,27 +97,47 @@ class Map extends React.Component {
   _setMapRegionAsync = async regionType => {
     let location = await Location.getCurrentPositionAsync({});
 
-    this.setState({
-      [regionType]: {
-        latitude: location.coords.latitude,
-        longitude: location.coords.longitude,
-        longitudeDelta: 0.005,
-        latitudeDelta: 0.005
-      }
-    });
+    const newMapRegion = {
+      latitude: location.coords.latitude,
+      longitude: location.coords.longitude,
+      longitudeDelta: 0.005,
+      latitudeDelta: 0.005
+    }
+
+     this.setState({
```

Comment:
I just put up a pr that takes map region out of local state and only stores it in the store which i think is a slightly cleaner way of doing this but take a look and lmk what you think